winch/environ/src/lib.rs

```diff
@@ -31,11 +33,33 @@ impl<'a> winch_codegen::FuncEnv for FuncEnv<'a> {
             index,
         }
     }
+
+    fn vmnative_call_host_func_native_call(&self) -> u8 {
+        let pointer_size = self.isa.triple().pointer_width().unwrap().bytes();
+        pointer_size.vmnative_call_host_func_context_func_ref()
+            + pointer_size.vm_func_ref_native_call()
+    }
+
+    fn vmctx_vmfunction_import_wasm_call(&self, index: u32) -> u32 {
+        let func_index = FuncIndex::from_u32(index);
+        self.offsets.vmctx_vmfunction_import_wasm_call(func_index)
+    }
+
+    fn vmctx_vmfunction_import_vmctx(&self, index: u32) -> u32 {
+        let func_index = FuncIndex::from_u32(index);
+        self.offsets.vmctx_vmfunction_import_vmctx(func_index)
+    }
```

Comment:
Out of curiosity, is there any particular reason to have accessor methods like this as opposed to exposing the `VMOffsets` directly?